### PR TITLE
Release/v0.32.11

### PR DESCRIPTION
**Pull request by melekes**

### CODE REVIEW

File: privval/retry_signer_client.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	return nil
+}
+
+func (sc *RetrySignerClient) SignVote(chainID string, vote *types.Vote) error {
+	for i := 0; i < sc.retries || sc.retries == 0; i++ {
+		err := sc.next.SignVote(chainID, vote)
+		if err == nil {
+			return nil
+		}
+		time.Sleep(sc.timeout)
```
Comment (marbar3778) [inline privval/retry_signer_client.go:63]: should there be logs here to tell the node operator that something is wrong

Comment (melekes) [reply privval/retry_signer_client.go:63]: we're logging errors inside `SignerClient`

File: node/node.go [hunk 2]
```diff
@@ -1192,7 +1192,19 @@ func createAndStartPrivValidatorSocketClient(
...
 
-	return pvsc, nil
+	// try to get a pubkey from private validate first time
+	pubKey := pvsc.GetPubKey()
+	if pubKey == nil {
+		return nil, errors.New("could not retrieve public key from private validator")
+	}
+
+	const (
+		retries = 50 // unlimited
```
Comment (melekes) [inline node/node.go:1202]:
```suggestion
		retries = 50 // 50 * 100ms = 5s total
```

**Comment by marbar3778** [comment]
is this good to be merged?

**Comment by melekes** [comment]
yep

**Pull request merged**